test/integration/integration_test.go

```diff
@@ -55,6 +55,16 @@ func TestMain(m *testing.M) {
 	var godogPaths = flag.String("paths", "./features", "")
 	flag.Parse()
 
+	if *godogTags != "" {
+		*godogTags += "&&"
+	}
+	runner := util.MinishiftRunner{CommandPath: givenPath}
+	if runner.IsCDK() {
+		*godogTags += "~minishift"
+	} else {
+		*godogTags += "~cdk"
+	}
```

Comment:
@praveenkumar No, `~` is used in godog to exclude tagged scenarios.

I am using it this way due to fact that as whole feature is tagged `@basic` I am not able to add `@minishift` tagged scenarios, they are already in. And so I am here excluding `@minishift` scenarios for CDK use and vice versa.

This way we are not forced to repetively tag every scenario cdk or minishift, but we tag just few of them which we want to exclude. But this is at cost of less readability.